tmt/steps/provision/mrack.py

```diff
@@ -358,6 +359,12 @@ class BeakerGuestData(tmt.steps.provision.GuestSshData):
         )
 
     # Guest request properties
+    whiteboard: Optional[str] = field(
+        default=None,
+        option=('-w', '--whiteboard'),
+        metavar='WHITEBOARD',
+        help='Whiteboard of beaker job.'
```

Comment:
The help message does not say much, what about something like `Text describing the job, displayed in the list of jobs` or similar?